source/_components/switch.recswitch.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Ankuoo Rec Switch"
+logo: ankuoo_recswitch.png
+description: "home-assistant.io web presence"
+date: 2018-10-05 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: "0.80"
+ha_category: Switch
+ha_iot_class: "Local Polling"
+ha_qa_scale: silver
```

Comment:
Authors should not decide on their own scale. Since this is a new component, please remove this for now.